src/screens/SearchScreen.js

```diff
@@ -1,49 +1,190 @@
 import { color } from "@rneui/base";
-import { Image , Platform, Text, View} from "react-native";
+import { Image, Platform, Text, View } from "react-native";
 import { SearchBar } from "react-native-elements";
 import { SafeAreaView } from "react-native-safe-area-context";
 import { useNavigation } from "@react-navigation/native";
 import { createStackNavigator } from "@react-navigation/stack";
 import { Button } from "react-native-elements";
-import { useState } from "react";
-import { Pressable } from "react-native";
+import { useState, useEffect} from "react";
+import { Pressable, FlatList, Item} from "react-native";
+import {supabase} from '../utils/hooks/supabase';
+import { SmallChatFill } from "../../assets/snapchat/NavigationIcons";
+
+
 const Stack = createStackNavigator();
 
-export default function SearchScreen(){
-    const navigation = useNavigation();
-    const [searchQuery, setSearchQuery] = useState('');
-return(
-    <View styles={{alignItems:"center"}}>
-    <SafeAreaView>
-        <View style={{flexDirection:"row", justifyContent:"space-between", gap:10, alignItems:"center", paddingLeft:10, paddingRight:10}}>
-        <SearchBar
-        containerStyle={{flex:1, width:100}}
-        width = "100"
-        placeholder="Search"
-        lightTheme = "true"
-        round = "true"
-        value={searchQuery}
-        onChangeText={setSearchQuery}
-        />
-        <Pressable
-        onPress={() =>{
-            navigation.navigate("UserTab");
-        }}
+export default function SearchScreen() {
+  const navigation = useNavigation();
+  const [searchQuery, setSearchQuery] = useState("");
+  const [usersToAdd, setUsersToAdd] = useState([]);
+
+  const [recents, setRecents] = useState([]);
+
+  useEffect(() => {
+      async function fetchUsers() {
+      try {
+          const { data, error } = await supabase
+          .from('usersToAdd')
```

Comment:
Looks like you made a new table on supabase! Please document this so others can set it up